frontend/tailwind.config.js

```diff
@@ -32,6 +38,11 @@ export default {
         'dark-theme-background': '#092e40',
         'dark-theme-foreground': '#25a5e3',
         'footer-background': '#181818',
+        'login-background': '#232323',
+        'input-background': '#F8F8F8',
+        'light-gray': '#686868',
+        'light-white': '#f9fafb',
+        'disabled-gray': '#3C3C3C',
       },
```

Comment:
Do we actually need all these? Can't we leverage tailwind classes ?

Reply:
Done.